README.md

```diff
@@ -154,15 +195,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+savraj = Person.new(name: "Savraj Singh", age: 24)
 ```
 
 Write Ruby code that will query for any person that is 15 years of age.
 
 Your answer...
 
 ```ruby
-Replace this with your answer
+Person.find(age: 15)
```

Comment:
:+1: although this will only find one person with age = 15. `.where` would return all of them